package.json

```diff
@@ -5,24 +5,25 @@
   "author": "Andres Rodriguez (joyent.com)",
   "private": true,
   "dependencies": {
-    "libuuid": "0.1.2",
+    "amqp": "git://github.com/postwait/node-amqp.git#2a5b1e9d2cd5838991ea378251f186c13ba79f97",
```

Comment:
`npm` has been reordering dependencies on `--save` for a while now, so I haven't done anything myself to sort this list alphabetically. If that's not something we want, please let me know.